rancher-common/rke.tf

```diff
@@ -4,14 +4,39 @@
 resource "rke_cluster" "rancher_cluster" {
   cluster_name = "quickstart-rancher-server"
 
-  nodes {
-    address          = var.node_public_ip
-    internal_address = var.node_internal_ip
-    user             = var.node_username
-    role             = ["controlplane", "etcd", "worker"]
-    ssh_key          = var.ssh_private_key_pem
+  dynamic nodes {
+    for_each = var.rancher_nodes == null ? [1] : []
+    content {
+      address          = var.node_public_ip
+      internal_address = var.node_internal_ip
+      user             = var.node_username
+      role             = ["controlplane", "etcd", "worker"]
+      ssh_key          = var.ssh_private_key_pem
```

Comment:
pass the key in file() e.g, :file((var.ssh_private_key_pem)

Reply:
fixed